f8a_worker/workers/githuber.py

```diff
@@ -87,6 +87,23 @@ def _get_topics(self):
 
         return topics
 
+    def _get_licenses(self):
+        """
+        Get license from GitHub repository through GitHub REST API
+
+        :return: name of repository license
+        """
+        if not self._repo_name:
+            return []
+
+        # GitHub returns always just one license per repository
+        license = requests.get(
+            "https://api.github.com/repos/{repo_name}/license".format(repo_name=self._repo_name)).json()
+        if license['message'] == 'Not Found':
+            return []
```

Comment:
Why are you returning array here and above? If I understand it correctly, github returns a string (one license string)?

Reply:
great catch :-D I copy-pasted topics and did not think that much about it. I will fix it